src/digital_identity_service/share.session.result.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const Validation = require('../yoti_common/validation');
+
+/**
+ * The share session result
+ *
+ * @class ShareSessionResult
+ */
+module.exports = class ShareSessionResult {
+  /**
+   * @param {Object} response
+   */
+  constructor(response) {
+    Validation.isString(response.id, 'Session ID');
+    this.id = response.id;
+
+    Validation.isString(response.status, 'Status');
+    this.status = response.status;
+
+    Validation.isString(response.expiry, 'Expiry');
+    const expiryDate = new Date(response.expiry);
+    if (expiryDate.toString() === 'Invalid Date') throw TypeError('Expiry must be a date like string');
```

Comment:
Given we are going to have other payload with date, let's move that to the utils.